micro-application-register/src/main/java/com/oath/micro/server/application/registry/Job.java

```diff
@@ -30,6 +31,12 @@
     private final String resourcePath;
     private final RegistryHealthChecker checker;
     private final RegistryStatsChecker statsChecker;
+    @Getter
+    private int scheduled =0;
+    @Getter
+    private int sent =0;
```

Comment:
I don't see this being used anywhere, as in no getSent() method call or geterrors()?

Reply:
will remove